pages/secret-manager/concepts.mdx

```diff
@@ -109,12 +109,18 @@ Secret types refer to the different kinds of sensitive data you can store with S
 
 Upon secret creation, you must choose a secret type that will also be applied to the secret version. All the secret's subsequent versions must be of the same type.
 
+## Scheduled deletion
+
+Scheduled deletion lets you mark a secret and its versions for deletion ahead of time. Instead of immediate deletion, the secret enters a 7-day pending deletion period, during which you can still [recover it](/secret-manager/how-to/recover-secrets/).
+
+During this time, you can read secret versions but cannot edit, access, or delete them. After the retention period, the secret and all its versions are permanently deleted. You can also schedule a secret version for deletion without impacting the secret associated with the version.
+
+Recovering deleted secrets is billed €0.01 per version associated with the secret.
```

Comment:
Would it be better to link to the pricing page, to avoid having binding mentions of price in the doc? E.g. "There is a small charge for recovering deleted secrets: you are billed per version associated with the secret. See the [pricing page](link) for detailed pricing information".